realm/realm-library/src/androidTest/java/io/realm/RealmQueryTest.java

```diff
@@ -1049,6 +1092,28 @@ public void testAverageValueForAllNullColumns() {
         assertEquals(0d, query.average(NullTypes.FIELD_DOUBLE_NULL), 0d);
     }
 
+    // Test average on columns with partial null rows
```

Comment:
This description doesn't seem to match the unit test (seems like copy paste from below?)